src/mpyl/cli/build.py

```diff
@@ -295,11 +307,20 @@ def ask_for_input(ctx, _param, value) -> Optional[str]:
     type=Pipeline(),
     required=False,
 )
+@click.option(
+    "--version",
+    "-v",
+    help="A specific version on https://pypi.org/project/mpyl/ to use for the build.",
+    type=click.STRING,
+    required=False,
+)
 @click.option(
     "--test",
```

Comment:
I think this name might be confusing to people? but `use_test_version` is a bit long maybe, not sure what is better

Reply:
It was like this before as well though. I can do `--version 1.1.0` and `--test_version 123.1234` if you think that's better?

Reply:
Eh I think I'm going to leave it like this, it requires more checks to implement two times the version selection. Maybe you can suggest an improvement to the flag description if they are unclear?